src/resolvelib/reporters.py

```diff
@@ -22,3 +22,15 @@ def ending_round(self, index, state):
     def ending(self, state):
         """Called before the resolution ends successfully.
         """
+
+    def adding_requirement(self, requirement):
+        """Adding a new requirement into the resolve criteria.
+        """
```

Comment:
nit: Documentation string phasing consistency would be nice.

Reply:
Not a nit, it's an important point and one I've just not go round to. Will fix this.